src/components/InlineTooltip.js

```diff
@@ -29,7 +31,9 @@ var InlineTooltip = React.createClass({
         style: React.PropTypes.object,
         icon: React.PropTypes.string,
         iconStyle: React.PropTypes.object,
-        timeout: React.PropTypes.number
+        timeout: React.PropTypes.number,
+        node : React.PropTypes.node,
+        type: React.PropTypes.string
```

Comment:
it will be great to describe all possible values. React.PropTypes.oneOf(['..', '..'])